fe/fe-core/pom.xml

```diff
@@ -981,6 +981,10 @@ under the License.
                     <artifactId>elasticsearch</artifactId>
                     <groupId>org.elasticsearch</groupId>
                 </exclusion>
+                <exclusion>
```

Comment:
You can't remove it, it will affect ranger work
https://github.com/StarRocks/starrocks/pull/47199

Reply:
Looks like VULN might already be fixed in 2.5.0 of _ranger-plugins-common_ so will revert the change